src/Files.App/UserControls/SideBar/SideBarItem.cs

```diff
@@ -25,6 +25,7 @@ public sealed partial class SidebarItem : Control
 		private bool isPointerOver = false;
 		private bool isClicking = false;
 		private object? selectedChildItem = null;
+		private PointerUpdateKind pointerUpdateKind = PointerUpdateKind.Other;
```

Comment:
`pointerUpdateKind` should be defined as a parameter to `Clicked()`, not a field. As it is, trying to open a sidebar item with the Enter key after doing a middle click would open the item in a new tab.

Reply:
That would require adding parameters to a couple of other events as well, would you be okay with updating the field when the enter key is pressed instead?

Reply:
Using fields is not recommended as it may cause unexpected bugs. I think the following would be good.
* Add `pointerUpdateKind` parameter to `Clicked()` and `RaiseItemInvoked()`.
* `LeftButtonReleased` is passed as a parameter when the enter key is pressed and when `RaiseItemInvoked()` is called from `SidebarItemAutomationPeer.Invoke()`.